features/admin.application-roles.v1/api/application-roles.ts

```diff
@@ -0,0 +1,445 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Let's update the date in license header

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6765/commits/7ec09bcd7a193a975a2e5ee3665cf6f83ae43f1c